llvm/lib/DebugInfo/DWARF/DWARFVerifier.cpp

```diff
@@ -531,14 +538,12 @@ bool DWARFVerifier::handleDebugInfo() {
   unsigned NumErrors = 0;
 
   OS << "Verifying .debug_info Unit Header Chain...\n";
-  DObj.forEachInfoSections([&](const DWARFSection &S) {
-    NumErrors += verifyUnitSection(S);
-  });
+  DObj.forEachInfoSections(
```

Comment:
is this unrelated reformatting? Could you remove any instances of unrelated reformatting?